tests/test_query.py

```diff
@@ -432,6 +432,32 @@ def test_merge(self):
     self.assertEqual(self._q(q3), {"a": 5, "b": 2, "c": 3})
     self.assertEqual(self._q(q4), {"a": 'a', "b": 'b', "c": 'c'})
 
+  def test_reverse(self):
+    #arrays
+    list = [0,1,2,3,4,5,6,7,8,9,10]
+    self.assertEqual(self._q(query.reverse(list)), list[::-1])
+    self.assertEqual(self._q(query.reverse(query.reverse(list))), list)
+    self.assertEqual(self._q(query.reverse(list[:0])), [])
```

Comment:
I really love those python idiosyncrasies, but wouldn't be better the literal list `[10, 9, 8, ...]` here?  😊

Reply:
I also think literal lists are more readable here.